inc/compatibility/fixes/class-sliderrevolutionfix.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Slider Revolution Fix
+ *
+ * @package Pantheon
+ */
+
+namespace Pantheon\Compatibility\Fixes;
+
+/**
+ * Slider Revolution Fix
+ */
+class SliderRevolutionFix {
+
+
+	public static function apply() {
+		$_SERVER['SERVER_NAME'] = $_SERVER['HTTP_HOST'];
```

Comment:
Consider using `PANTHEON_HOSTNAME` instead, see: https://github.com/pantheon-systems/WordPress/blob/default/wp-config-pantheon.php#L99-L104

(Really 🔢 anytime `$_SERVER['HTTP_HOST']` would be used)